main.go

```diff
@@ -100,16 +90,15 @@ func entrypoint(shutdown chan os.Signal) {
 		defer mongoClient.Disconnect()
 	}
 	if err != nil {
-		log.WithFields(logrus.Fields{
-			"error": logrus.Fields{"message": err.Error()},
-		}).Errorf("failed router setup")
+		failLogger.routerSetup(err)
 		return
 	}
 	log.Trace("router setup completed")
 
 	srv := &http.Server{
-		Addr:    fmt.Sprintf("0.0.0.0:%s", env.HTTPPort),
-		Handler: router,
+		Addr:              fmt.Sprintf("0.0.0.0:%s", env.HTTPPort),
+		Handler:           router,
+		ReadHeaderTimeout: time.Millisecond * 5,
```

Comment:
Why this timeout? Why 5ms? it seems a bit too short. What is the rationale?

Reply:
It is just for ReadHeaderTimeout, is a mitigatin for a gosec:
https://github.com/rond-authz/rond/runs/6868594249?check_suite_focus=true

Reply:
Ok, why 5ms however?

Reply:
I thought it was enough to just parse headers, should I increase it?
I would avoid resource consuption...

Reply:
it's pretty strict I don't know if it is enough; I'd relax it a bit

Reply:
OK, increased to 1 sec